docs/dialect.md

```diff
@@ -173,3 +173,110 @@ Allowable date parts: year, quarter, month, dayofyear, day, weekday, week, hou
+
+  ```xml 
+    <format-bool-as-value value='TrueFirst' />
+  ```
+  
+ **Boolean Capabilities**  <br/>
+  
+  Boolean usage is also defined by capabilities in the manifest file. <br\>
+  See the Query section for boolean capabilities [here]({{ site.baseurl }}/docs/capabilities) for more details.
+
+ **TDVT Coverage**  <br/>
+The following TDVT tests check that the boolean functionality is working as expected for a connector. 
+logical.bool	`exprtests\standard\setup.logical.bool.txt` and logical	`exprtests\standard\setup.logical.txt`. See [this](https://github.com/tableau/connector-plugin-sdk/tree/master/tdvt/tdvt/exprtests/standard) for more details. 
+
+### Temporary Table Support:
+   **format-create-table** <br/> 
+  This function uses  piece-by-piece formulas for creating a table.
+  Predicates can be used with tokens that only corresponds to a certain type of table.
+  Predicates: <br/>
+  -  __GlobalTemp__
+  -  __LocalTemp__
+  -  __AnyTemp__
+  -  __NoTemp__ <br/>
+   You should also be able to use the following string substitution tokens along with the predicate:
+      - __%n - table name__
+     - __%f - formatted column list__
+    This function is avialable when `CAP_CREATE_TEMP_TABLES` capability is set to yes. 
+  
+  ```xml 
+    <format-create-table>
+        <formula>CREATE </formula>
+        <formula predicate='GlobalTemp'>GLOBAL </formula>
+        <formula predicate='LocalTemp'>LOCAL </formula>
+        <formula predicate='AnyTemp'>TEMPORARY </formula>
+        <formula>TABLE %n (</formula>
+        <formula>%f</formula>
+        <formula>)</formula>
+        <formula predicate='AnyTemp'> ON COMMIT PRESERVE ROWS</formula>
+      </format-create-table>
+  ```
+
+  **format-select** <br/>
+  This function uses piece-by-piece formula for defining a SELECT statement. Here, we can define the clause used in `SELECT` statement. <br/>
+  the `Into` clause in `SELECT` statement creates a new table. `<format-select>` will help you define how your TEMP table is created when using `INTO` clause.
+  `INTO` cluse is only aviable when `CAP_SELECT_INTO` capability is set to yes. 
+  
+  ```xml 
+      <format-select>
+        <part name='Into' value='CREATE GLOBAL TEMPORARY TABLE %1 ON COMMIT PRESERVE ROWS AS' />
+        <part name='Top' value='SELECT * FROM (' />
+        <part name='Select' value='SELECT %1' />
+        <part name='From' value='FROM %1' />
+        <part name='Where' value='WHERE %1' />
+        ...
+      </format-select> />
+  ```
+
+  **format-drop-table** <br/>
+    This function defines the format for dropping a table. %1 is the table name. Each formula is executed as a separate statement. <br/>
+    This function is only used when `CAP_TEMP_TABLES_NOT_SESSION_SCOPED` capability is set to yes. 
+  ```xml 
+     <format-drop-table>
+      <formula>TRUNCATE TABLE %1</formula>
+      <formula>DROP TABLE %1 PURGE</formula>
+    </format-drop-table>
+  ``` 
+  
+ **Temp Tables Capabilities**  <br/>
```

Comment:
Nit: Should probably be singular like the other capabilities sections, i.e. "Temp Table Capabilities"

Reply:
Made this change. Makes sense.